### PR TITLE
Fixed Linting errors

### PR DESCRIPTION
**Pull request by raghavpuri**
## What did you implement:

Fixed Linting errors

## How did you implement it:

changed small errors 

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by rehrumesh** [comment]
@raghavpuri there are merge conflicts to be resolved.

File: examples/compute/aws-ec2.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+
+const ncAWS = nodeCloud.getProvider("AWS", process.env.ncconf);
 const options = {
-  apiVersion: '2016-11-15',
+  apiVersion: "2016-11-15",
 };
 
-// get compute object for AWS
-const ec2 = ncAWS.compute(options);
+// get container object for AWS
```
Comment (rehrumesh) [inline examples/compute/aws-ec2.js]: We have a separate section for ecs. Why ec2 section has been changed?

Comment (rehrumesh) [reply examples/compute/aws-ec2.js]: @rajikaimal

**Comment by raghavpuri** [comment]
Sure @rehrumesh  i'll look into it

**Comment by raghavpuri** [comment]
@rehrumesh @charithccmc @agentmilindu  please review :)

File: examples/compute/aws-ecs.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
-const nodeCloud = require('../../lib/');
-const ncAWS = nodeCloud.getProvider('AWS', process.env.ncconf);
+const nodeCloud = require("../../lib/");
```
Comment (agentmilindu) [inline examples/compute/aws-ecs.js]: According to the Airbnb Style Guide, strings should be enclosed in `'`s, not `"`s. 

I see you have done this correctly in `examples/compute/aws-ec2.js` file. I'm wondering how the Style Guide changed suddenly.

**Review by agentmilindu — CHANGES_REQUESTED** [review]

**Comment by raghavpuri** [comment]
Made the changes :)

**Review by agentmilindu — APPROVED** [review]
LGTM

**Comment by agentmilindu** [comment]
@rehrumesh @rajikaimal Please review and merge.

**Review by rehrumesh — APPROVED** [review]

**Comment by rehrumesh** [comment]
Great work

**Comment by raghavpuri** [comment]
Thanks!

**Pull request merged**